src/main/assembly/dist/cfg/application.properties

```diff
@@ -3,6 +3,7 @@
 #
 deposits.inbox=
 deposits.auto-publish=true
+deposits.outbox=/var/opt/dans.knaw.nl/tmp/deposits-outbox
```

Comment:
@ChrisMuller80 : this is not according to specs. As a matter of fact, I am OK with this change in the specs, but you should include a motivated note in your cover message, so that in cases where we disagree about the change a discussion takes place.

Reply:
Noted